src/map/mob.cpp

```diff
@@ -5500,7 +5501,8 @@ uint64 MobSummonDatabase::parseBodyNode(const YAML::Node &node) {
 			this->invalidWarning(node["Default"], "Unknown mob %s.\n", mob_name.c_str());
 			return 0;
 		}
-		summon->default_mob_id = mob->vd.class_;
+
```

Comment:
i am not sure if we care for code style in this matter but we should avoid unnecessary empty lines.